caas/kubernetes/provider/bootstrap_test.go

```diff
@@ -764,7 +765,20 @@ mkdir -p $JUJU_TOOLS_DIR
 cp /opt/jujud $JUJU_TOOLS_DIR/jujud
 
 test -e $JUJU_DATA_DIR/agents/controller-0/agent.conf || JUJU_DEV_FEATURE_FLAGS=developer-mode $JUJU_TOOLS_DIR/jujud bootstrap-state $JUJU_DATA_DIR/bootstrap-params --data-dir $JUJU_DATA_DIR --debug --timeout 10m0s
-JUJU_DEV_FEATURE_FLAGS=developer-mode $JUJU_TOOLS_DIR/jujud machine --data-dir $JUJU_DATA_DIR --controller-id 0 --log-to-stderr --debug
+
+mkdir -p /var/lib/pebble/default/layers
+cat > /var/lib/pebble/default/layers/001-jujud.yaml <<EOF
+summary: jujud service
+services:
+    jujud:
+        summary: Juju controller agent
+        startup: enabled
+        override: replace
+        command: JUJU_DEV_FEATURE_FLAGS=developer-mode $JUJU_TOOLS_DIR/jujud machine --data-dir $JUJU_DATA_DIR --controller-id 0 --log-to-stderr --debug
```

Comment:
I'm not sure exactly what we're testing here so this may not be a problem (I haven't looked closely at the associated test), but this `command:` line is weird to me, because the command value isn't run via shell, so the `ENV=value` syntax isn't going to work -- Pebble treats the first argument as the command name for `os/exec`.

Reply:
yeah these lines are generated by the bootstrapping code, might have to change how that is done